airflow/config_templates/config.yml

```diff
@@ -1221,7 +1221,14 @@ openlineage:
       example: '{"type": "http", "url": "http://localhost:5000"}'
       default: ''
       version_added: ~
-
+    disable_source_code:
+      description: |
+        If disabled, OpenLineage events do not contain source code of particular
+        operators, like PythonOperator.
+      default: "True"
```

Comment:
Since this is disabled by default, I wonder if we should flip this into `add_source_code`.